bin/check-aggregate.rb

```diff
@@ -185,6 +185,32 @@ def acquire_aggregate
   def named_aggregate_results
     results = api_request("/aggregates/#{config[:check]}?max_age=#{config[:age]}")[:results]
     warning "No aggregates found in last #{config[:age]} seconds" if %w(ok warning critical unknown).all? { |x| results[x.to_sym] == 0 }
+
+    err_results = []
+    if results[:critical] > 0
```

Comment:
Also is there a reason this is not an if elsif block currently? I think the next if should be part of an if/elsif block unless I am missing something as I dont see the value in checking again.